pkgs/tools/audio/beets/default.nix

```
@@ -164,6 +169,11 @@ in pythonPackages.buildPythonApplication rec {
     nose
     rarfile
     responses
+    # The following are plugin's deps but there are tests for them so we add them
+    # to our input no matter what the user may override in his overlays.
```

Nitpick:

```suggestion
    # to our input no matter what users may override in their overlays.
```

Nevertheless, if the feature is explicitly disabled by the user, why even run the tests?

---

The point is, that beets includes tests for some of the plugins that depend on these libraries. In case a user disables a plugin through his overlays, these dependencies aren't included in the `buildInputs` but they are needed in the `checkPhase` for the tests to succeed.

If you'll find a way to tell beets not perform specifically these tests, that would be better.